### ISSUE TITLE
Would a satellite transmission work with the sdr? hackrf for example

### ISSUE THREAD

**Issue by j1m1l0k0** [fsphil/hacktv#104]
Is there any way to adapt the code for DVB-S2/S/T demodulation and sending via satellite?

**Comment by fsphil** [comment]
Do you mean can you use a DVB channel as an input for hacktv? If so then yes, using software like tvheadend or mumudvb - then pass the URL for your channel as an input to hacktv.

**Comment by j1m1l0k0** [comment]
It actually wasn't that.  I would like to know if there is a way to implement in source code demod dvb-s2, dvb-s, signals via satellite equal to dvb-s2 demod gui.  See this app https://forum.amsat-dl.org/index.php?thread/101-software-dvb-s-demodulator/

**Comment by fsphil** [comment]
No, that's outside the scope of the project. However the TS output from that program could be used as a source for hacktv.

**Comment by j1m1l0k0** [comment]
Thanks Phil ;)
